src/Culling/babylon.boundingBox.ts

```diff
@@ -1,4 +1,7 @@
-﻿module BABYLON {
+module BABYLON {
+
+    const _identityMatrix = Matrix.Identity();

```

Comment:
I added that as there was in the BoundingSphere.
it allows an optimization when the constructor or the `reConstruct` method is called wihtout no input worldMatrix as it avoid transforming the 8 `vectors` to compute `vectorsWorld `